tests/HttpClientInterception.Tests/HttpClientInterceptorOptionsTests.cs

```diff
@@ -24,7 +25,7 @@ public static async Task GetResponseAsync_Returns_Null_If_Request_Not_Registered
             var request = new HttpRequestMessage(HttpMethod.Get, "https://google.com");
 
             // Act
-            HttpResponseMessage actual = await options.GetResponseAsync(request);
+            HttpResponseMessage actual = await options.GetResponseAsync(request, CancellationToken.None);
```

Comment:
The need to change so many tests makes me thing the token being an optional parameter might be the better approach to reduce source-breaking changes in 3.0.0.